index.html

```diff
@@ -0,0 +1,383 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>KMarketer</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+    <!-- Header -->
+    <header class="header">
+        <div class="header__container">
+            <h1 class="logo">KMarketer</h1>
+            <div class="header__icons">
+                <img src="./assets/search-icon.png" alt="Search Icon" class="header__search-icon">
```

Comment:
you can just give the following icons the same name, cz i think they have the same styles